libs/language-server/src/stdlib/builtin-blocktypes/LocalFileExtractor.jv

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+/**
+* Extracts a `File` from the local storage.
+* 
+* @example Fetches a file from the given local path.
+* block CarsFileExtractor oftype LocalFileExtractor {
+*   filePath: "cars.csv";
+* }
+*/
+builtin blocktype LocalFileExtractor {
+	input default oftype None;
+	output default oftype File;
+	
+	/**
+	* The PATH to the file in the local storage to extract.
```

Comment:
Why `PATH`? There is a `PATH` env variable on basicaly any operating system so this is needlessly confusing. Just go with `path`. Same comment about `localstorage` as for the block description as well.

```suggestion
	* The path to the file in the local file system to extract.
```